getting-started-tutorial/getting-started-tutorial.ipynb

```diff
@@ -1556,8 +1549,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "project.set_function('hub://describe', 'describe')\n",
-    "describe = project.func('describe').apply(mount_v3io())"
+    "describe = project.set_function('hub://describe', 'describe').apply(mount_v3io())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### View the function documentation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "View the function description and parameters by calling the `doc` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "function: describe\n",
+      "describe and visualizes dataset stats\n",
+      "default handler: summarize\n",
+      "entry points:\n",
+      "  summarize: Summarize a table\n",
+      "    context(MLClientCtx)  - the function context, default=\n",
+      "    table(DataItem)  - MLRun input pointing to pandas dataframe (csv/parquet file path), default=\n",
+      "    label_column(str)  - ground truth column label, default=None\n",
+      "    class_labels(List[str])  - label for each class in tables and plots, default=[]\n",
+      "    plot_hist(bool)  - (True) set this to False for large tables, default=True\n",
+      "    plots_dest(str)  - destination folder of summary plots (relative to artifact_path), default=plots\n",
+      "    update_dataset  - when the table is a registered dataset update the charts in-place, default=False\n"
```

Comment:
I hadn't reviewed this embedded MLRun API doc yet, and I don't like it. It doesn't fit the conventions that I had already started applying when I worked on the MLRun docs (such as beginning the descriptions with a capital letter) and there are grammar issues, etc. Perhaps we can omit this output until a later MLRun/MRLun Functions version with updated doc? We can try to prioritize the doc review for the `describe` function.

Reply:
I think that if we need to work with what we have. I'd like people to know that .doc() exists and that they can use that. Once we get to address the description of the marketplace functions, it will be fixed for all users automatically.